alvr/client_core/cpp/gamma_pass.cpp

```diff
@@ -17,10 +17,23 @@ const string PASSTHROUGH_FRAGMENT_SHADER = R"glsl(#version 300 es
         uniform samplerExternalOES tex0;
         in vec2 uv;
         out vec4 color;
+
+        const float div12 = 1. / 12.92;
+        const float div1 = 1. / 1.055;
+        
+        float srgbToLinear(float val)
+        {
+          return val < 0.04045
+          ? val * div12
+          : pow((val + 0.055) * div1, 2.4);
```

Comment:
This operation feels a bit weird. I think it can either be converted to [MAD](https://www.khronos.org/opengl/wiki/GLSL_Optimizations#Get_MAD), or simplified.

Reply:
Yes you can definately optimize this, I know I wrote the code... https://github.com/korejan/ALVR-OpenXR-Engine/blob/12ff6a4162f4f13c71e6f26029769ca85844ac05/src/alxr_engine/vulkan_shaders/common/sRGBLinearize.glsl

Reply:
you can't do MAD here because the channels need to be if'd separately.

Reply:
> you can't do MAD here because the channels need to be if'd separately.

No, I didn't mean to remove the condition, but optimize `(val + 0.055) * div1`.

Reply:
cool well then why don't you make a PR then @korejan, since yours is already optimized

Reply:
if i were to optimize mine then you'd come at me for it also being optimized

Reply:
If i was here some minute earlier I would have made the same comment about linearization lol. But you know what? Give Korejan their damn plug. As long as the code is efficient. But Korejan you need to stop now alright?

Reply:
@korejan it's all yours!

Reply:
Actually @galister, if you let me I would try to make a PR myself with the code even more optimized...

Reply:
Sure thing. Feel free to close the PR.